source/documentation/glossary.md

```diff
@@ -99,6 +99,24 @@ Case management system used by Crown Courts only to record case details, hearing
 
 ## Other technical terms
 
+### Audience
+
+**Internal** means the data should not be shared outside the department.
```

Comment:
We should possibly be a bit clearer here. There are times when internal data can be shared with other departments, when the required governance has been completed and agreements are in place. Wondering if we might be able to get a line from the data protection team?